core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/toDataFrame.kt

```diff
@@ -141,7 +144,26 @@ internal fun convertToDataFrame(
         val property = it
         if (excludes.contains(property)) return@mapNotNull null
 
+        class ValueClassConverter(val unbox: Method, val box: Method)
+
+        val valueClassConverter = (it.returnType.classifier as? KClass<*>)?.let { kClass ->
+            if (!kClass.isValue) return@let null else {
```

Comment:
Why do we need explicit return here? This seems readable enough:
```
if (!kClass.isValue) null else {
```

Reply:
right, thank you